### PR TITLE
some style changes 

### PR DESCRIPTION
**Pull request by mayagao**
address some of the issues i created

### CODE REVIEW

File: site/navigation.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ class Navigation extends React.Component {
       if (!items || items.length === 0) return null;
       return items.map((member) => {
         let linkContainerClasses;
-        let linkClasses = 'txt-link mr12 mr0-mm inline-block block-mm txt-s';
+        let linkClasses = 'txt-link dark-link mr12 mr0-mm inline-block block-mm txt-s';
```
Comment (samanpwbb) [inline site/navigation.js]: you can use `txt-link--{anycolor}` to set the link color, the `dark-link` class is not necessary.

File: src/typography.css [hunk 2]
```diff
@@ -547,6 +547,10 @@ textarea {
   transition: color var(--transition);
 }
 
+.txt-link.dark-link {
```
Comment (samanpwbb) [inline src/typography.css]: 🔥

File: src/typography.css [hunk 3]
```diff
@@ -560,6 +564,11 @@ textarea {
   color: var(--blue-dark);
 }
 
+.txt-link.dark-link:hover {
```
Comment (samanpwbb) [inline src/typography.css]: 🔥

File: site/documentation/entry.js [hunk 4]
```diff
@@ -71,19 +71,20 @@ class Entry extends React.Component {
...
           </div>
           <div className='inline' id={`collapsed-${selectors && selectors[0]}`}>
             {collapsedSelectorEls}
           </div>
           {expandButton}
-        </div>
-        <div className='col col--12 col--8-ml'>
-          <div className='mb48 prose'>
+
+          <div style={{ fontSize: '13px', lineHeight: '20px' }} className='mt12 mb18 alpha75 txt-s'>
```
Comment (samanpwbb) [inline site/documentation/entry.js:83]: Inline styling like this defeats the purpose of a css framework. if you aren't happy with our typographic scale, make an issue about it. Not going to merge this.

**Comment by mayagao** [comment]
ok lets close

**Pull request closed without merging**